src/components/@shared/FormInput/index.tsx

```diff
@@ -118,16 +121,17 @@ export default function Input(props: Partial<InputProps>): ReactElement {
             *
           </span>
         )}
-        {help && <Tooltip content={<Markdown text={help} />} />}
+        {help && !prominentHelp && (
+          <Tooltip content={<Markdown text={help} />} />
+        )}
       </Label>
       <InputElement size={size} {...field} {...props} />
-
+      {help && prominentHelp && <FormHelp>{help}</FormHelp>}
```

Comment:
this now pushes down all error messages:

<img width="927" alt="Screen Shot 2022-04-20 at 15 28 21" src="https://user-images.githubusercontent.com/90316/164254138-eb5f327d-c455-4440-a2ad-9a0fa7068f00.png">

As help text can be long or short we are at the limit of hacking it in place with `position: absolute` like I did here with the default provider link: https://github.com/oceanprotocol/market/blob/v4/src/components/%40shared/FormFields/Provider/index.module.css#L10-L11

So we need to figure out a new layout so that error messages are always attached to input field, but components added before or after it won't push it down